hack/update-generated-defaulters.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
```

Comment:
Add `hack/update-generated-defaulters.sh, hack/update-generated-listers.sh, hack/update-generated-openapi.sh` to `make update` section.